test/run_unit.sh.in

```diff
@@ -114,6 +117,9 @@ else
 	echo "============================================================="
 	echo ":test-result: FAIL $tst" >> $test_output
 fi
-exit 1
+
+if test "z$fail_fast" == "ztrue"; then
+    exit 1
+fi
```

Comment:
@vmiklos, this addresses your point about stopping after first failure. Agreed that it's not preferable by default, so made it optional.